tests/features/package-explorer.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Package Explorer - View details about Package
+  User can navigate to and view details of what TPA knows about individual Package
+
+  Background:
+    Given There is SBOM with Package details ingested
+
+  Scenario: View Package metadata
+    When User visits Package details page
+    # like from SBOM details page - Packages tab
+    Then Package name should be visible in metadata section
+    * Package version should be visible in metadata section
```

Comment:
Can we have the astrix as "And" keyword?

Reply:
Switch '*' to And.

Multiple 'then' I used when the validation block has different meaning - different logical set of tests (here just for separating/highlighting the Pagination vs specific fields).